pymc/distributions/continuous.py

```diff
@@ -1642,6 +1642,15 @@ def logp(value, b, kappa, mu):
             msg="b > 0, kappa > 0",
         )
 
+    def icdf(p, mu, b, kappa):
```

Comment:
```suggestion
    def icdf(value, b, kappa, mu):
```
To keep consistent with the signatures of other `icdf` methods and the signatures of methods under this class